pyproject.toml

```diff
@@ -52,7 +53,7 @@ allow-ambiguous-features = true
 
 [project.optional-dependencies]
 pulser = ["pulser>=v0.15.2", "pasqal-cloud>=0.3.5"]
-braket = ["amazon-braket-sdk"]
+braket = ["amazon-braket-sdk<1.72.2"]
```

Comment:
Why caping the version ?

Reply:
1.72.2 introduces a check on the parameter name that we fail in some cases

Reply:
i also added the pin to `all`